value/arguments/api.graphql

```diff
@@ -0,0 +1,44 @@
+# A `@value` directive defines a value that can be applied in various contexts
```

Comment:
Snippets are meant to be small, concise examples, but here we have multiple examples of the same concept.

Can we just simplify to a single example?

Reply:
Think the point was missed, simplify to a single/small number of constant examples, not put multiple fields in the same snippet.